src/aioquic/quic/congestion/cubic.py

```diff
@@ -0,0 +1,206 @@
+from .base import (
+    K_MINIMUM_WINDOW,
+    K_INITIAL_WINDOW,
+    QuicCongestionControl,
+    QuicRttMonitor,
+    register_congestion_control,
+)
+from ..packet_builder import QuicSentPacket
+from typing import Iterable, Dict, Any
+
+# cubic specific variables (see https://www.rfc-editor.org/rfc/rfc9438.html#name-definitions)
+K_CUBIC_K = 1    
+K_CUBIC_C = 0.4
+K_CUBIC_LOSS_REDUCTION_FACTOR = 0.7
+K_CUBIC_MAX_IDLE_TIME = 2   # reset the cwnd after 2 seconds of inactivity
+
+def better_cube_root(self, x):
```

Comment:
self? While we are at it, please add type annotations otherwise `mypy` will fail